erpnext/erpnext_integrations/custom/contact.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Why is the `is_billing_contact` field exported as fixtures or a custom field. It should be in standard doctype right? Please fix and reopen

Reply:
@deepeshgarg007 I have that PR before and it is advised to have on ERPNext PR. As other changes on this does not cover frappe scope. It is more on ERPNext related scope.

Here is my PR before https://github.com/frappe/frappe/pull/8840

See comment here https://github.com/frappe/frappe/pull/8840#issuecomment-558469686